lib/ansible/plugins/inventory/docker_swarm.py

```diff
@@ -63,6 +63,9 @@
                          The port always defaults to M(2376).
             type: bool
             default: no
+        include_host_uri_port:
+            description: Override the detected port number included in I(ansible_host_uri)
+            type: str
```

Comment:
Shouldn't this be an `int` (and converted to a string when needed)?

Reply:
(TBH, I'm not sure whether inventory plugins actually support that...)

Reply:
I was wondering what is safer. After all, it will be string at the end. I am not sure of here the type reference is just for documentation or it really means something. I just merge it as any other string but was wandering on some additional checkings

Reply:
I think it is documentation only. See https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/__init__.py#L55-L62 (I've been checking it out for #53906.)

Reply:
I may change it to in the, I see no difference

Reply:
At least then nobody can claim that they were allowed to put an arbitrary string in there ;)